### ISSUE TITLE
Explain limiting request scope

### ISSUE THREAD

**Issue by DavidSchinazi** [tfpauly/draft-age-masque-connect-ip#6]
The protocol contains a mechanism that allows limiting the scope of the request, but doesn't explain why it would be desirable to do so. Can we add text explaining when a client would want to limit itself, and how a server can leverage this information?

**Comment by tfpauly** [comment]
Sure. I think the examples provide some insight here, but we should spell it out.

**Comment by mirjak** [comment]
I still think that reasoning about this as a differ mode of operation helps. Depending on the use case, you either always or never want to limit the scope of your requests. There might be use case where both is used but I think the common mode of operation might be doing one or the other. In any case I agree that providing more reasoning is useful. Maybe this is part of an applicability section?

**Comment by gloinul** [comment]
Yes, I think explaining a bit for what mechanism makes sense in which usages would help a lot.
